examples/matmul_automove.py

```diff
@@ -72,11 +72,13 @@ async def run_matmul():
                         memsize += b_block.nbytes
```

Comment:
We have no clear plan for now. That's what Yineng and I talked about through Slack. But the values should be back to outer array (original array) because other tasks may need it. For example, our blocked cholesky has a bunch of data dependencies among tasks. If we cannot push back the result to the outside, we cannot make cholesky work. I will raise this issue to Friday meeting.

Reply:
Yeah.. this is tricky. We should definitely gather opinions from Mattan/George/Chris/etc about how this should behave.